app/config/locale/translations/ne.json

```
@@ -1,32 +1,32 @@
 {
-    "settings.inspire": "\"बुद्धिमान हुनु को कला के लाई बेवास्ता गर्न जान्ने को कला हो।\"",
+    "settings.inspire": "\"के लाई बेवास्ता गर्ने भन्ने जान्नुनै बुद्धिमान हुनुको कला हो ।\"",
     "settings.locale": "ne",
     "settings.direction": "ltr",
-    "emails.sender": "%s टीम",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.sender": "%s समूह",
+    "emails.verification.subject": "खाता प्रमाणिकरण",
+    "emails.verification.hello": "नमस्ते {{name}}",
+    "emails.verification.body": "इमेल ठेगाना प्रमाणित गर्नको लागी यो लिंकमा जानुहोस।",
+    "emails.verification.footer": "यदि तपाइँले आफ्नो खाता प्रमाणित गर्न सोध्नु भएको छैन भने तपाइँले यो सन्देश लाई बेवास्ता गर्न सक्नुहुन्छ।",
+    "emails.verification.thanks": "धन्यवाद",
+    "emails.verification.signature": "{{project}} समूह",
+    "emails.magicSession.subject": "लगइन",
+    "emails.magicSession.hello": "नमस्ते,",
+    "emails.magicSession.body": "लगइन गर्नको लागी यो लिंकमा जानुहोस।",
+    "emails.magicSession.footer": "यदि तपाइँले यो इमेल प्रयोग गरेर लगइन गर्न सोध्नु भएको छैन भने तपाइँले यो सन्देश लाई बेवास्ता गर्न सक्नुहुन्छ।",
+    "emails.magicSession.thanks": "धन्यवाद",
+    "emails.magicSession.signature": "{{project}} समूह",
+    "emails.recovery.subject": "पासवर्ड रिसेट",
+    "emails.recovery.hello": "नमस्ते {{name}}",
+    "emails.recovery.body": "{{project}}को पासवर्ड रिसेट गर्नको लागी यो लिंकमा जानुहोस।",
```

```suggestion
    "emails.recovery.body": "{{project}} को पासवर्ड रिसेट गर्नको लागी यो लिंकमा जानुहोस।",
```

Please keep the templates consistent with the English translations file and add the missing spaces in the other strings as well

---

@christyjacob4 In the Nepali language, space doesn't make sense there.